contrib/patterns/_grid-media.scss

```diff
@@ -0,0 +1,52 @@
+$medium-screen: 1000px;
```

Comment:
Sadly that causes my fun little trick not to work, so i'm leaving this the way it is.